cg/services/fastq_file_service/fastq_file_service.py

```diff
@@ -31,3 +32,19 @@ def concatenate(
 
         if temp_reverse:
             temp_reverse.rename(reverse_output)
+
+    @staticmethod
+    def get_concatenated_forward_read_output_path(fastq_directory: Path, sample_name: str) -> Path:
+        return get_concatenated_read_output_path(
+            fastq_directory=fastq_directory,
+            sample_name=sample_name,
+            direction=ReadDirection.FORWARD,
+        )
+
+    @staticmethod
+    def get_concatenated_reverse_read_output_path(fastq_directory: Path, sample_name: str) -> Path:
+        return get_concatenated_read_output_path(
+            fastq_directory=fastq_directory,
+            sample_name=sample_name,
+            direction=ReadDirection.REVERSE,
+        )
```

Comment:
And then if we move the path logic to this class, this could be also moved to utils 🎈

Reply:
Yes!